spec.html

```diff
@@ -25176,9 +25176,9 @@ <h1>Runtime Semantics: Evaluation</h1>
         <emu-alg>
           1. Let _value_ be ? BindingClassDeclarationEvaluation of |ClassDeclaration|.
           1. Let _className_ be the sole element of BoundNames of |ClassDeclaration|.
-          1. If _className_ is *"\*default\*"*, then
+          1. If _className_ is ~default~, then
             1. Let _env_ be the running execution context's LexicalEnvironment.
-            1. Perform ? InitializeBoundName(*"\*default\*"*, _value_, _env_).
+            1. Perform ? InitializeBoundName(~default~, _value_, _env_).
```

Comment:
InitializeBoundName only accepts strings

Reply:
ah, i just pulled the commit in untouched. i'll revert this line.

Reply:
I think you just need to extends the Assert at the top of InitializeBoundName to include `~default~`.